src/main/java/seedu/address/logic/commands/OpenCommand.java

```diff
@@ -0,0 +1,72 @@
+package seedu.address.logic.commands;
+
+import static seedu.address.logic.parser.CliSyntax.PREFIX_TAG_NAME;
+
+import java.awt.Desktop;
+import java.io.File;
+import java.io.IOException;
+
+import javafx.collections.transformation.FilteredList;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.tag.Tag;
+import seedu.address.model.tag.TagName;
+
+public class OpenCommand extends Command {
+
+    public static final String COMMAND_WORD = "open";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Opens the file specified in the filepath of a tag. "
+            + "\nParameters: "
```

Comment:
Is the "\n" before parameters intended? All the other command does not use "\n" before parameters, I think it is better to make it consistent.

Reply:
I find that without "\n" the format is very hard to read, so I added them.

Reply:
Okay then. Should we do that to all other commands?

Reply:
That's going to be more readable in my opinion. 
Right now the tag command is like this. You cannot distinguish tag name and file address easily.
![image](https://user-images.githubusercontent.com/51979232/95745940-90f68680-0cc8-11eb-9eca-aa728cf1282e.png)

Reply:
After seeing the screenshot, I also think it is better to use \n since we have so much space for the text box. Noted!